packages/react-instantsearch-core-next/src/__tests__/InstantSearch.test.tsx

```diff
@@ -81,7 +81,7 @@ describe('InstantSearch', () => {
 
     render(<InstantSearch indexName="indexName" searchClient={searchClient} />);
 
-    // @TODO once props are stable:
+    // @TODO: figure out why it's called 6 times instead of 3.
     // expect(searchClient.addAlgoliaAgent).toHaveBeenCalledTimes(3);
```

Comment:
I need to figure out why it's called 6 times instead of 3. Any ideas yet?

Reply:
the other three tiems are:

```js
  [
      [ 'instantsearch.js (4.30.0)' ],
      [ 'react (16.14.0)' ],
      [ 'react-instantsearch (7.0.0-alpha.0)' ],
      [ 'react-instantsearch-hooks (7.0.0-alpha.0)' ],
      [ 'JS Helper (3.5.5)' ],
      [ 'JS Helper (3.5.5)' ]
    ]
```

not really sure how to enforce that as those other calls are also valid (and the client deduplicates anyway). We could assert those other calls too, or not consider it important there's 3 (and not more) calls

Reply:
maybe we could make the helper mocked for this test (in a separate file)?

Reply:
Not sure it's necessary if we don't care how many times it's called. Just ensuring the calls with React, React InstantSearch and React InstantSearch Hooks should be enough to cover what we mean to assert here.